### PR TITLE
Viewports

### PR DESCRIPTION
**Pull request by bdenham**
## Description

This PR adds topics for Page Builder viewports and associated features. It also reorganizes things to improve the overall structure of the documentation. 

## Viewport topics Ready for Review
- docs/viewports/how-viewports-work.md
- docs/viewports/how-to-add-viewports.md
- https://devdocs.magedevteam.com/149/page-builder/docs/viewports/how-viewports-work.html
- https://devdocs.magedevteam.com/149/page-builder/docs/viewports/how-to-add-viewports.html

## Viewport topics In-Progress
- docs/viewports/how-to-use-viewports.md
- docs/viewports/how-to-change-viewports.md
- https://devdocs.magedevteam.com/149/page-builder/docs/viewports/how-to-use-viewports.html
- https://devdocs.magedevteam.com/149/page-builder/docs/viewports/how-to-change-viewports.html

### CODE REVIEW

File: docs/viewports/how-to-add-viewports.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# How to add viewports
+
+By default, Page Builder defines four viewports, but only two are shown on the Admin stage: `desktop` and `mobile`. The other two viewports (hidden from the stage), are `tablet` and `small-mobile`. This topic shows you how to add the hidden viewports to the stage and customize them as needed.
```
Comment (omiroshnichenko) [inline docs/viewports/how-to-add-viewports.md:3]: I would say it defines four breakpoints and use two of them for viewports

Comment (bdenham) [reply docs/viewports/how-to-add-viewports.md:3]: Ah, good. I've gone back and forth on the use of viewports vs. breakpoints...good catch.

File: docs/viewports/pagebuilder-viewports.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+            </var>
+        </var>
+    </var>
+</var>
+```
+
+Descriptions of each node follows:
+
+- `Magento_PageBuilder` - Defines the module to which viewport data applies.
+- `breakpoints` - Defines `viewports` object for the module.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:206]: Defines breakpoints

File: docs/viewports/pagebuilder-viewports.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+                </var>
+            </var>
+        </var>
+    </var>
+</var>
+```
+
+Descriptions of each node follows:
+
+- `Magento_PageBuilder` - Defines the module to which viewport data applies.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:205]: Defines module view config

File: docs/viewports/pagebuilder-viewports.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+- `breakpoints` - Defines `viewports` object for the module.
+- `desktop` and `mobile` - Defines viewport objects named `desktop` and `mobile`.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:207]: Defines breakpoint objects

File: docs/viewports/pagebuilder-viewports.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+- `label` - (string) Defines title of tooltip.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:208]: Used for viewport description

File: docs/viewports/pagebuilder-viewports.md [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+- `stage` - (bool) Determines whether viewport button appears above the stage.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:209]: Enable viewport on stage for current breakpoint

File: docs/viewports/pagebuilder-viewports.md [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+- `default` - (bool) Determines if the viewport is selected when the stage is loaded.
+- `class` - (string) CSS class to apply to the viewport switcher button (not the stage).
+- `icon` - (string) - URL to the button image (SVG) location.
+- `media` - (string) Media query used in the Admin.
+- `conditions` - Defines the `min` and `max` breakpoints for the viewport.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:214]: Defines breakpoint conditions

File: docs/viewports/pagebuilder-viewports.md [hunk 8]
```diff
@@ -0,0 +1,329 @@
...
+- `max-width` - (string) Defines the maximum width for the viewport in `px`.
+- `min-width` - (string) Defines the minimum width for the viewport in `px`.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:216]: Defines with for breakpoint

File: docs/viewports/pagebuilder-viewports.md [hunk 9]
```diff
@@ -0,0 +1,329 @@
...
+- `options` - Viewport data defined for the `Products` content type. You can define similar nodes with unique names for your own content types.
+- `products` - Viewport data defined for the `Products` content type. You can define similar nodes with unique names for your own content types.
+- `default` - Viewport data defined for the `default` appearance of the `Products` content type. You can define similar nodes with unique names for your own content types.
+- `slidesToShow` - Viewport data to set the `slidesToShow` variables within the `preview.js` and `widget.js` components of the `Products` content type. You can define similar data with unique names for your own content types.
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:220]: Config for Products widget for current breakpoint

File: docs/viewports/pagebuilder-viewports.md [hunk 10]
```diff
@@ -0,0 +1,329 @@
...
+
+As shown for the `Products` content type, you can define your own viewport data hierarchy to fit your needs. For example, if you have a custom content type called `testimonials`— which showed customer quotes similar to how the `Products` carousel works — you could create a new node in the `mobile` viewport called `viewport-data` node, as shown here:
```
Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:222]: define your own breakpoint data

Comment (omiroshnichenko) [inline docs/viewports/pagebuilder-viewports.md:222]: in the `mobile` breakpoint

**Review by omiroshnichenko — APPROVED** [review]
To summarize, Viewport is what we see on the stage, Breakpoint is what we use to configure  Viewports and place where configure our storefront widgets behaviour base on different window sizes

**Pull request merged**